app/scss/styles.scss

```diff
@@ -0,0 +1,65 @@
+* {
```

Comment:
maybe you will have to divide this classes into separate files according to it's usage